src/prefect/cli/flow_run.py

```diff
@@ -135,3 +136,42 @@ async def cancel(id: UUID):
         )
 
     exit_with_success(f"Flow run '{id}' was succcessfully scheduled for cancellation.")
+
+
+@flow_run_app.command()
+async def logs(id: UUID):
+    """
+    View logs for a flow run.
+    """
+    page_size = 200
+    offset = 0
+    more_logs = True
+    log_filter = LogFilter(flow_run_id={"any_": [id]})
+
+    async with get_client() as client:
+        # Get the flow run
+        try:
+            flow_run = await client.read_flow_run(id)
+        except ObjectNotFound as exc:
+            exit_with_error(f"Flow run {str(id)!r} not found!")
+
+        while more_logs:
+            # Get the next page of logs
+            page_logs = await client.read_logs(
+                log_filter=log_filter, limit=page_size, offset=offset
+            )
+
+            # Print the logs
+            for log in page_logs:
+                app.console.print(
+                    # Print following the flow run format (declared in logging.yml)
+                    f"{pendulum.instance(log.timestamp).to_datetime_string()}.{log.timestamp.microsecond // 1000:03d} | {logging.getLevelName(log.level):7s} | Flow run {flow_run.name!r} - {log.message}",
+                    soft_wrap=True,
+                )
+
+            # Wait for the user to press enter to get the next page
```

Comment:
This PR no longer includes user input, though, correct?

Reply:
Correct. As discussed with @madkinsz , we omitted it, and we only load and print the pages chunk by chunk.

Reply:
I would think we don't need that comment on line 172 then

Reply:
```suggestion
```